cmd/apps.go

```diff
@@ -16,9 +16,26 @@ import (
 	"github.com/deis/workflow-cli/settings"
 )
 
+// AppCmd is interface definition for running app commands
+type AppCmd interface {
```

Comment:
I think it's the same amount of code in `cmd`, but in deis.go we'd need to have an instance of every struct available there rather than one massive struct that implements the entire `cmd` package. Either way works for me though, just a heads up